docs/Spark-Parquet-Data-Loading.md

```diff
@@ -0,0 +1,94 @@
+`org.cbioportal.persistence.spark.util.ParquetConstants.java` contains all file names and directories for Parquet files. 
+
+All files should be placed in PARQUET_DIR `${data.parquet.folder}` property specified in the [properties](portal.properties-Reference.md#spark-parquet) with **case_lists**, **gene_panels**, **studies** as the top level directories.
+
+### Example of Organization
+ 
+* PARQUET_DIR
+    * **case_lists**
+        * msk_impact_2017_cna
+    * **gene_panels**
+        * impact341
+    * **studies**
+        * msk_impact_2017
+            * clinical_sample
+            * mutations
+
+### Parquet File Name
+
+| TSV file                         	                                     | Parquet file                                                           	        | Parquet Type  |
+|------------------------------------------------------------------------|----------------------------------------------------------------------------------|---------------|
+| data_clinical_patient*.txt                                             | studies/{study_id}/clinical_patient                                              | data          |
+| data_clinical_sample*.txt                                              | studies/{study_id}/clinical_sample                                               | data          |
+| data_cna_*.seg<br>(e.g. data_cna_hg19.seg)                             | studies/{study_id}/data_cna_*.seg<br>(e.g. studies/{study_id}/data_cna_hg19.seg) | data          |
+| data_mutations*.txt                                                    | studies/{study_id}/mutations                                                     | data          |
+| data_fusions*.txt                                                      | studies/{study_id}/fusions                                                       | data          |
+| data_CNA*.txt                                                          | studies/{study_id}/data_CNA                                                      | cna           |
+| meta*.txt                                                              | studies/{study_id}/meta                                                          | meta          |
+| case_list/*.txt<br>(e.g. cases_cna.txt)                                | case_lists/{stable_id}<br>(e.g. case_list/msk_impact_2017_cna)                   | case          |
+| data_gene_panel_{panel_id}.txt <br>(e.g. data_gene_panel_impact341.txt) | gene_panels/{panel_id}<br>(e.g. gene_panels/impact341)                           | panel         |
+
+**Note:** *.seg* files are kept in separate folders in order to preserve `reference_genome_id`.
+ 
+## Writing Parquet Files
+
+ParquetWriter takes arguments:<br>
+**--input-file** `<path to the TSV file>`<br>
+**--output-file** `<path to write Parquet file>`<br>
+**--input-file-type** `<type of input file - see previous Parquet File Name table>`<br>
```

Comment:
Since the header in the table is called "Parquet Type" I would change the description here -

"<Parquet Type - see Parquet File Name table>"